python/satyaml/S-NET_A.yml

```diff
@@ -1,5 +1,5 @@
 name: S-NET A
-norad: 43188
+norad: 43186
```

Comment:
Are you sure about this? The DK3WN reference doesn't say whether S-NET A is 43186. Celestrak lists 43188 for S-NET A.

Reply:
I reverted the changes on these files, they were right before